src/main/java/io/github/opencubicchunks/cubicchunks/core/network/WorldEncoder.java

```diff
@@ -167,7 +186,7 @@ static int getEncodedSize(Chunk column) {
     static int getEncodedSize(Collection<Cube> cubes) {
         int size = 0;
 
-        size += 2 * cubes.size(); // 1. isEmpty and hasStorage flags
+        size += 3 * cubes.size(); // 1. isEmpty, hasStorage and hasBiomeArray flags
```

Comment:
Already adding network incompatible change, might be a good idea to encode more boolean flags into byte[], potentially also giving a way to add future changes. It's small amount of data so probably not really worth it as an optimization, but it would allow to add 5 new flags later while still allowing new clients to join old servers.